tests/telemetry/validate_yang_events.py

```diff
@@ -0,0 +1,81 @@
+import yang as ly

```

Comment:
How do you install this package to sonic-mgmt container?

Reply:
This script is not being run in sonic-mgmt container, it is being run inside duthost. Please look at line 78 in test_events.py

Reply:
Why not use sonic_yang for yang validation?

Reply:
I tried but parsing fails because my input json is not in config db format. So I used libyang directly since sonic_yang is just a wrapper.

Reply:
We are going to upgrade libyang, if the API is changed, you'll have to update your implementation.

Reply:
I am using same API as: https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py#L171. If API is changed, I will update appropriately.